mindsdb_native/libs/helpers/text_helpers.py

```diff
@@ -148,18 +148,16 @@ def hashtext(cell):
 
 
 def _is_foreign_key_name(name):
-    for endings in ['-id', '_id', 'ID', 'Id']:
-        if name.endswith(endings):
-            return True
+    for endings in ['id', 'ID', 'Id']:
+        for add in ['-','_', ' ', '']:
```

Comment:
That last `''` element is not needed, is it?

Reply:
Hmh, good point, the approach I have here is a bit... ahm, stupid.

Reply:
Ok, see new approach, it's closer to what I'd have wanted to achieve.